packages/mui-material/src/Alert/Alert.js

```diff
@@ -191,6 +191,7 @@ const Alert = React.forwardRef(function Alert(inProps, ref) {
   const classes = useUtilityClasses(ownerState);
 
   const externalForwardedProps = {
+    ...other,
```

Comment:
Why is `...other` here?

Reply:
So that the `useSlot` can handle the `component` prop (in `…other`) properly for the root slot.

Reply:
We don't do it like this in any other component (AFAIK). We're destructuring `component` and passing it manually, like in https://github.com/mui/material-ui/pull/44729/files#diff-3f734f0cc98a1307bcc5136c215b2399af94e61664776e6927d2a5a153ce2f61 for example.

I think adding the entire `other` here is prone to bugs in the future.

Reply:
Make sense.